### PR TITLE
Spellchecking with wmllint

### PR DESCRIPTION
**Pull request by cooljeanius**
So, I've been running `wmllint` some more for spellchecking, and adding any custom spellings it catches to its lists of recognized spellings. This isn't all of them, but it should be the most repeated terms, at least.

### CODE REVIEW

File: _main.cfg [hunk 1]
```diff
@@ -27,6 +28,8 @@
 	[/modify_unit_type]
 	{./credits.cfg}
 	icon=data/add-ons/Black_Cross_of_Aleron/images/units/elves-wood/initiate/attack2.png~RC(magenta>teal)
+	# wmllint: general spelling merfolk orcish fae ghast nightgaunt mayhaps
```
Comment (CelticMinstrel) [inline _main.cfg:31]: Is it required to have spellings grouped above a string? Couldn't they all be gathered in a block above the `[campaign]` tag?

I think I've never actually checked spelling with `wmllint` – probably I didn't have `pyenchant` installed. But it certainly can't hurt to have these comments.

Comment (cooljeanius) [reply _main.cfg:31]: Yeah I think they can be put anywhere; putting them here just made the most intuitive sense to me... I can move it, though.

Comment (cooljeanius) [reply _main.cfg]: ok moved in c389ea7

File: scenarios/07-dread-marsh.cfg [hunk 2]
```diff
@@ -973,7 +974,7 @@
 		[/filter]
 		[popup_message]
 			image=portraits/humans/necromancer.png
-			message=_"The ghast lord Krrgra falls, collapsing into a puddle of slime that rapidly soaks into the ground and vanishes into the stagnant waters." + "
+			message=_"The ghast lord Krrrgra falls, collapsing into a puddle of slime that rapidly soaks into the ground and vanishes into the stagnant waters." + "
```
Comment (CelticMinstrel) [inline scenarios/07-dread-marsh.cfg:977]: Oh hey it actually found a misspelling, cool.

File: scenarios/10-back-home.cfg [hunk 3]
```diff
@@ -761,11 +762,12 @@
 						[/and]
 					[/show_if]
 					speaker=tw_1st
+					#wmllint: local spelling Thrar
 					message=_"If Mary is a human name... and Siegel is a human surname... I wonder... could they both refer to the same person? Thrar, do you know of a Mary Siegel?"
 				[/message]
 				[message]
 					speaker=dwarfhermit
-					message=_"Hmmm, cannae say that I do, lassie. I know a few Siegels, but no Mary amongst 'em. But I cannae say that there isnae one."
+					message=_"Hmmm, cannae say that I do, lassie. I know a few Siegels, but no Mary amongst 'em. But I cannae say that there isnae one." #wmllint: no spelcheck
```
Comment (CelticMinstrel) [inline scenarios/10-back-home.cfg:770]: Typo in the directive

File: scenarios/subscen/10b-final-malcorn.cfg [hunk 4]
```diff
@@ -202,6 +202,7 @@
 		[/message]
 		[message]
 			id=chiefguard
+			#wmllint: local spelling Aphrophila
```
Comment (CelticMinstrel) [inline scenarios/subscen/10b-final-malcorn.cfg:205]: Is this really the only place her name appears? I would've thought she'd be a directory spelling instead…

Not really complaining though.

**Comment by CelticMinstrel** [comment]
All those "no spellcheck" lines are kinda unfortunate, especially in the cases where they exist because of just one non-word, like a scream or groan… I do understand that using them on the Scots-ish dwarven talk is probably necessary though.

File: scenarios/10-back-home.cfg (hunk 3 above)
Comment (cooljeanius) [reply scenarios/10-back-home.cfg:770]: lol

File: scenarios/subscen/10b-final-malcorn.cfg (hunk 4 above)
Comment (cooljeanius) [reply scenarios/subscen/10b-final-malcorn.cfg:205]: It's the only time another character addresses her by name; looks like `wmllint` skipped checking when it was initially declared in her `name=` key...

File: scenarios/07-dread-marsh.cfg (hunk 2 above)
Comment (cooljeanius) [reply scenarios/07-dread-marsh.cfg:977]: 3, actually!

File: scenarios/subscen/10b-final-malcorn.cfg (hunk 4 above)
Comment (cooljeanius) [reply scenarios/subscen/10b-final-malcorn.cfg]: ok, moved to a directory spelling in e73e33f

File: scenarios/10-back-home.cfg (hunk 3 above)
Comment (cooljeanius) [reply scenarios/10-back-home.cfg]: fixed in e73e33f

**Pull request merged**